test-workspace/src/main/scala/example/User.scala

```diff
@@ -1,6 +1,23 @@
 package example
 
+case class User2(name: String, age : Int = 12)
+
+
+
 object Main {
```

Comment:
Sorry for the conflicts here, we should probably keep a gentleman's agreement to never change this file to avoid conflicts. It's probably best to use a local copy of this workspace in a different external directory.

Reply:
I plan on changing it back, I just didn't manage yesterday. I agree we should probably not change this.